table/block_based/block_based_table_builder.cc

```diff
@@ -283,7 +295,7 @@ struct BlockBasedTableBuilder::Rep {
   const BlockBasedTableOptions table_options;
   const InternalKeyComparator& internal_comparator;
   WritableFileWriter* file;
-  std::atomic<uint64_t> offset;
+  uint64_t offset = 0;
```

Comment:
Here we access a variable from multiple threads. Although in x86, this should be equal to atomic variable with relaxed memory order, it's not guaranteed in other platforms, so it's considered unsafe. As what the code base does for other variables. these variables are always made atomic, even if only relaxed memory order is used.